mkgenesis/Makefile

```
@@ -1,4 +1,4 @@
-all: balances.txt.gz README.md
+all: balances.txt.gz README.md clean
```

```suggestion
all: balances.txt.gz README.md
```

we don't want to run clean by default